django_plastic_tickets/locale/de/LC_MESSAGES/django.po

```diff
@@ -32,91 +32,103 @@ msgstr ""
 msgstr "Dichte in g/cm^3"
 
-#: models.py:89
+#: django_plastic_tickets/models.py:89
 msgid "A RAL classic color number, or 0 if a color can not be specified"
 msgstr ""
 "Eine RAL klassik Farbnummer, oder 0 wenn eine Farbe nicht spezifiert werden "
 "kann"
 
-#: models.py:98
+#: django_plastic_tickets/models.py:98
 msgid "None"
 msgstr "Keine"
 
-#: models.py:134
+#: django_plastic_tickets/models.py:134
 msgid "Optimal nozzle temperature"
 msgstr "Optimale Nozzle temperatur"
 
-#: models.py:136
+#: django_plastic_tickets/models.py:136
 msgid "Optimal bed temperature"
 msgstr "Optimale Druckbett temperatur"
 
-#: models.py:138
+#: django_plastic_tickets/models.py:138
 msgid "Diameter of the filament"
 msgstr "Durchmesser des Filaments"
 
-#: models.py:146
+#: django_plastic_tickets/models.py:146
 msgid "Weight of the empty package in gram"
 msgstr "Gewicht der leeren Verpackung in gram"
 
-#: models.py:149
+#: django_plastic_tickets/models.py:149
 msgid "Amount of material in gram, which is usually shipped with this package"
 msgstr ""
 "Menge des Materials in gram, das normalerweise mit dieser Verpackung "
 "versandt wird"
 
-#: models.py:152
+#: django_plastic_tickets/models.py:152
 msgid "Model name of the spool, as defined by the manufacturer"
 msgstr "Modell name der Spule, wie vom Hersteller angegeben"
 
-#: models.py:154
+#: django_plastic_tickets/models.py:154
 msgid "Short description of the appearance of this package"
 msgstr "Kurze beschreibung des Aussehens der Verpackung"
 
-#: models.py:173
+#: django_plastic_tickets/models.py:173
 msgid "The internal label to identify the material"
 msgstr "Die interne Kennzeichnung, um das Material zu indentifizieren"
 
-#: models.py:176
+#: django_plastic_tickets/models.py:176
 msgid ""
 "Initial weight of the material in gram, not including the weight of the "
 "package"
 msgstr ""
 "Anfängliches gewicht des Materials in gram, ohne das gewicht der Verpackung"
 
-#: models.py:179
+#: django_plastic_tickets/models.py:179
 msgid ""
 "Current weight of the material in gram, including the weight of the package"
 msgstr "Aktuelles gewicht des Materials in gram, einschließlich der Verpackung"
 
-#: models.py:182
+#: django_plastic_tickets/models.py:182
 msgid "Total price, including the package."
 msgstr "Preis einschließlich der Verpackug"
 
-#: models.py:185
+#: django_plastic_tickets/models.py:185
 msgid "The owner of the material"
 msgstr "Der Besitzer des Materials"
 
-#: models.py:188
+#: django_plastic_tickets/models.py:188
 msgid "Hints about how to use this material"
 msgstr "Hinweise bezüglich der verwendung des Materials"
 
-#: templates/plastic_tickets/new_ticket.html:40
+#: django_plastic_tickets/models.py:208
+msgid "Open"
+msgstr "Offen"
+
+#: django_plastic_tickets/models.py:209
+msgid "Closed"
+msgstr "Geschlossen"
+
+#: django_plastic_tickets/models.py:218
+msgid "The staff member who works on the ticket"
+msgstr "Die Mitarbeiter*in, die das Ticket bearbeitet"
```

Comment:
I'd rather like to use a [gender colon](https://de.wikipedia.org/wiki/Gender-Doppelpunkt) here - mostly because of the screen reader argument.